docs/UserGuide.md

```diff
@@ -491,7 +491,7 @@ Displayed after command: `summary`
 |------------------|------------------------------------------------------------------------------------------------------------------------------|
 | **Add**          | `add id/STUDENT_ID n/NAME e/EMAIL g/GRADE [gp/GROUP]...` <br> e.g., `add id/A01234567X n/James Ho e/e0123456@u.nus.edu g/B+` |
 | **Delete**       | `delete STUDENT_ID`<br> e.g., `delete A0123456X`                                                                             |
-| **Edit**         | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [gp/GROUP]...` <br> e.g.,`edit A0123466C g/A+`                   |
+| **Edit**         | `edit STUDENT_ID [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [g/GRADE] [gp/GROUP]...` <br> e.g.,`edit A0123466C g/A+`                   |
```

Comment:
I think we dont have address and phone number anymore for the params.

Reply:
We can also edit the student_id.

Reply:
edit STUDENT_ID [id/STUDENT_ID] [n/NAME] [e/EMAIL] [g/GRADE] [gp/GROUP_NAME]...

Reply:
I think the issue should be fixed now